src/main/java/com/cupid/jikting/common/util/PasswordUtil.java

```diff
@@ -0,0 +1,28 @@
+package com.cupid.jikting.common.util;
+
+import java.util.Random;
+import java.util.stream.IntStream;
+
+public class PasswordUtil {
```

Comment:
p1. 유틸성 클래스들은 외부에서 객체를 생성하지 못하도록 생성자를 막아주세요!

```suggestion

@NoArgsConstructor(access = AccessLevel.PROTECTED)
public class PasswordUtil {
```